compiled/main.css.map

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@nykka Do we need this file?

Reply:
This file is automatically created when run `rake sass`